bigquery/tests/unit/test_client.py

```diff
@@ -23,6 +23,14 @@
 import six
 from six.moves import http_client
 import pytest
+try:
+    import pandas
+except (ImportError, AttributeError):  # pragma: NO COVER
```

Comment:
A bit strange that AttributeError is required. Is this from debugging the 2.7 issue?

Reply:
I saw it like this in `test_job.py` and copied it. It was added in https://github.com/GoogleCloudPlatform/google-cloud-python/pull/4585/commits/c72ba75da1cbd62f89fcf88ef0a36178b13de9d5. Should I add to anywhere we do pandas/pyarrow imports?